sessionctx/variable/tidb_vars.go

```diff
@@ -183,6 +183,12 @@ const (
 	// If the query has a LIMIT clause, high concurrency makes the system do much more work than needed.
 	TiDBDistSQLScanConcurrency = "tidb_distsql_scan_concurrency"
 
+	// TiDBTwoPhaseCommitterConcurrency is used to set the concurrency of for phase committer.
+	TiDBTwoPhaseCommitterConcurrency = "tidb_two_phase_committer_concurrency"
```

Comment:
```suggestion
	TiDBCommitterConcurrency = "tidb_committer_concurrency"
```
I think it is better to make it simple.